pypxe-server.py

```diff
@@ -117,7 +119,7 @@
                 args.NETBOOT_FILE = 'boot.http.ipxe'
 
         #serve all files from one directory
-        os.chdir (args.NETBOOT_DIR)
+        os.chdir(args.NETBOOT_DIR)
```

Comment:
That is true. We really ought to be passing the directory into the module for both TFTP and HTTP, which is supported but we currently are not doing. This is OK as it is then.